.changeset/popular-mangos-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/react": patch
```

Comment:
if this is a new feature this should be `minor`.

Reply:
Sorry I messed up, my first contribution to OSS. It is closed unintentionally, I've discarded my changes and performed again, should I reopen it or what?

Reply:
no worries, yeah reopening should be fine.